src/backend/chat.nim

```diff
@@ -75,15 +75,18 @@ proc sendChatMessage*(
     }
   ])
 
-proc sendImages*(chatId: string, images: var seq[string]): RpcResponse[JsonNode] {.raises: [Exception].} =
+proc sendImages*(chatId: string, images: var seq[string], msg: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # Add "placeholder" text to instead of empty string as mobile app
+  let text = if (msg == ""): "<p>placeholder</p>" else: msg
```

Comment:
Mobile sends `<p>placeholder</p>` in case of no text? That sounds weird.

Reply:
> I'm not a fan of this `<p>placeholder</p>` thing. Can you confirm that mobile does this and maybe also ask why they do it? It seems like something is not getting handled correctly in status-go if we have to send bogus text. It's also not great for the network.

Unfortunately, we can't send images without text (as I remember it was a limitation in status-go). I think it was a reason why mobile app add a placeholder text (we did the same when add emoji). My opinion, we should send several messages as one message + text (with several image Urls). In that case its makes more sense and easy to implement in QML.
For now, it's too expansive every time when we receive a new message check every each message and connect it with others.

Reply:
@borismelnik i just sent an image without text fine using release 0.10.0

Reply:
@borismelnik do you know if fixing status-go is a possibility? I just don't like that both mobile and desktop have to send exactly the same placeholder text. If there is a discrepancy, we will show the placeholder to users, which is bad.

As for sending all images in a single message with one text, do it if you can now, but again, the UI part (new design) is not part of this issue. We'll probably do it in 0.12.

Reply:
@jrainville we can track it in another ticket suitable for any of other milestones. I think happy path is satisfied here so we can address everything else when implementing UI separately (https://github.com/status-im/status-desktop/issues/6374 is a good one to be put as candidate for 0.12)